### PR TITLE
[AN-4520] Adds new preference category to options settings

### PR DESCRIPTION
**Pull request by shumengye**
#### Description

Adds new section to options settings with toggles for send button and dark theme

#### Ticket
 https://wearezeta.atlassian.net/browse/AN-4520
#### APK
[Download build #7630](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7630/artifact/build/artifact/wire-dev-PR182-7630.apk)
[Download build #7633](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7633/artifact/build/artifact/wire-dev-PR182-7633.apk)
[Download build #7634](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7634/artifact/build/artifact/wire-dev-PR182-7634.apk)
[Download build #7637](http://192.168.10.18:8080/job/Pull%20Request%20Builder/7637/artifact/build/artifact/wire-dev-PR182-7637.apk)

### CODE REVIEW

**Comment by shumengye** [comment]
build again

File: wire-core/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -480,8 +480,9 @@
     <string name="pref_options_image_download_always">Always</string>
     <string name="pref_options_image_download_wifi">Wi-Fi only</string>
     <string name="pref_options_media_category_title">Media</string>
+    <string name="pref_options_requested_category_title">By popular demand</string>
     <string name="pref_options_cursor_send_button_title">Send button</string>
-    <string name="pref_options_cursor_send_button_summary">Show send button</string>
+    <string name="pref_options_cursor_send_button_summary">Disable to send via the keyboard.</string>
```
Comment (hakonbo) [inline wire-core/src/main/res/values/strings.xml]: Is this the final copy, sounds a bit awkward.

Comment (hakonbo) [inline wire-core/src/main/res/values/strings.xml]: Is this the final copy? Sounds a bit awkward.

**Comment by hakonbo** [comment]
Also a minor bug where send button is visible even though it's disabled in settings. Steps to reproduce:
1. Make sure setting is to show send button
2. Open a conversation, write some text and make sure send button is visible, do not send but leave the text there
3. Go to settings and disable send button
4. Return to conversation. Now the send button will be visible all the time.

File: wire-core/src/main/res/values/strings.xml (hunk 1 above)
Comment (shumengye) [inline wire-core/src/main/res/values/strings.xml]: Yes this is the final copy from Roger via @Yorgos-V

**Pull request merged**